.github/workflows/ci-npm-publish.yml

```diff
@@ -58,8 +58,8 @@ jobs:
       node_auth_token: ${{ secrets.CF_NPM_JS_PACKAGES_READ_WRITE }}
```

Comment:
lets use `--tag ${{ needs.get-release-context.outputs.publish-tag }}` for the cli publishing command as well